### DISCUSSION TITLE
problem with focus() in CTkEntry

### DISCUSSION THREAD

**Discussion by hadete22** [TomSchimansky/CustomTkinter#1061] [Q&A]
Problem: self.myField.focus() is not working, but on original tkinter it is work. Don`t know why.

myW = customtkinter.CTk()
l1=customtkinter.CTkEntry(myW)
l1.grid(row=1, columnspan=1)
l1.focus()
myW.mainloop()

**Comment by Akascape** [comment]
@hadete22 Try `focus_set()`
```python
l1.focus_set()
```

**Reply by hadete22** [reply]
No, I tried

**Comment by hadete22** [comment]
I found that if `I create myW as Tk, not as CTk`
And then create CTkEntry and add it on myW, `it works`

import tkinter
import customtkinter
myW = tkinter.Tk()
l1=customtkinter.CTkEntry(myW)
l1.grid(row=1, columnspan=1)
l1.focus_set()
myW.mainloop()
